gulpfile.js

```diff
@@ -13,18 +12,20 @@ sass.compiler = require("node-sass");
 function throwError(e) {
     throw new Error("sass compilation failed", e);
 }
-module.exports = {
-    scss: function() {
-        return src("**/*.scss", "!**/example/**")
+
+module.exports = function(gulp) {
+    gulp.task("build", function() {
+        return gulp
+            .src("**/*.scss", "!**/example/**")
             .pipe(sass({ importer }).on("error", throwError))
-            .pipe(dest("./"));
-    },
-    css: function() {
-        return src(["**/*.css", "!**/example/**", "!**/*.min.css"])
             .pipe(postcss([autoprefixer()]))
-            .pipe(dest("./"))
+            .pipe(gulp.dest("./"))
             .pipe(postcss([cssnano()]))
             .pipe(rename({ suffix: ".min" }))
-            .pipe(dest("./"));
-    },
+            .pipe(gulp.dest("./"));
+    });
+    gulp.task("build:watch", function() {
+        return gulp.watch("**/*.scss", gulp.series("build"));
```

Comment:
Should we exclude the `example` folder here as well? It won't get built either way

Reply:
Yes. But the syntax exlude syntax actually did not do anything. Its fixed now.